packages/fx-core/src/component/coordinator/index.ts

```diff
@@ -325,6 +209,25 @@ class Coordinator {
         } else {
           warnings = res.value.warnings;
         }
+      } else if (isNewGeneratorEnabled()) {
+        const generators = Generators.filter((g) => g.activate(context, inputs));
```

Comment:
Use `find` instead of `filter` will be more efficient and easier to understand.

Reply:
Updated, thanks~